proto/oracle/query.proto

```diff
@@ -196,6 +202,18 @@ message QueryAggregateVotesResponse {
   repeated AggregateExchangeRateVote aggregate_votes = 1 [(gogoproto.nullable) = false];
 }
 
+// QuerySlashWindow is the request type for the
+// Query/SlashWindow RPC method.
+message QuerySlashWindow {}
```

Comment:
Based on the above naming pattern, we should name this as QuerySlashWindowRequest right?

Reply:
ah yes good point, I'll fix